api.planx.uk/inviteToPay/sendPaymentEmail.ts

```diff
@@ -8,8 +8,9 @@ const sendSinglePaymentEmail = async (
 ) => {
   try {
     const payment = await validatePaymentRequest(paymentRequestId, template);
-    if (!payment || Boolean(payment?.paid_at)) throw Error(`Invalid payment request: ${paymentRequestId}`);
-
+    if (!payment) throw Error("Unable to send payment email, cannot find payment request");
```

Comment:
this addresses lingering feedback from #1592, moving validation checks (eg is it already paid) to the `validatePaymentRequest` method below